test/system/conversations_test.rb

```diff
@@ -3,11 +3,40 @@
 class ConversationsTest < ApplicationSystemTestCase
   setup do
     @conversation = conversations(:greeting)
+    @assistant = assistants(:keith_gpt4)
     login_as @conversation.user
   end
 
   test "visiting the index" do
     visit conversations_url
     assert_selector "h1", text: "Conversations"
   end
+
+  test "creating new chat with meta+shift+o" do
+    visit conversation_messages_path(@conversation)
+
+    page.driver.browser.action.key_down(:meta)
+                              .key_down(:shift)
+                              .send_keys("o")
+                              .key_up(:meta)
+                              .key_up(:shift)
+                              .perform
+
+    expected_path = new_assistant_message_path(@assistant)
+
+    assert_current_path(expected_path)
+  end
+
+  test "creating new chat with meta+j" do
+    visit conversation_messages_path(@conversation)
+
+    page.driver.browser.action.key_down(:meta)
+                              .send_keys("j")
+                              .key_up(:meta)
+                              .perform
+
```

Comment:
There is a test helper method for sending keystrokes so you don’t need to grab the page object. I believe send_keys? Search for that

I use it for the other keyboard short cuts which all have tests. It lets you use the same naming as the stimulus key names